README.md

```diff
@@ -1,72 +1,71 @@
-# GNUPLOTASS2
-
-#-------------------------------------------------------------------------------------------------------------------
-
-## Circle.h
-### Constructor, Destructor, member function and member variable declarations
-
-## Ellipse.h
-### Constructor, Destructor, member function and member variable declarations
-
-## Line.h
-### Constructor, Destructor, member function and member variable declarations
-
-## Point.h
-### Constructor, Destructor, member function and member variable declarations
-
-## Rectangle.h
-### Constructor, Destructor, member function and member variable declarations
-
-## Shapes2D.h
-### Constructor, Destructor, member function and member variable declarations
-### Parent class and pure virtual function declarations
-
-## Square.h
-### Constructor, Destructor, member function and member variable declarations
-
-## Triangle.h
-### Constructor, Destructor, member function and member variable declarations
-
-#-------------------------------------------------------------------------------------------------------------------
-
-## Circle.cpp
-### Actual calculation for Plot_Corrdinate() function is done
-### X & Y coordinate values get printed in respected txt file
-
-## Ellipse.cpp
-### Actual calculation for Plot_Corrdinate() function is done
-### X & Y coordinate values get printed in respected txt file
-
-## Line.cpp
-### Actual calculation for Plot_Corrdinate() function is done
-### X & Y coordinate values get printed in respected txt file
-
-## Rectangle.cpp
-### Actual calculation for Plot_Corrdinate() function is done
-### X & Y coordinate values get printed in respected txt file
-
-## Square.cpp
-### Actual calculation for Plot_Corrdinate() function is done
-### X & Y coordinate values get printed in respected txt file
-
-## Triangle.cpp
-### Actual calculation for Plot_Corrdinate() function is done
-### X & Y coordinate values get printed in respected txt file
-
-#-------------------------------------------------------------------------------------------------------------------
-
-## main.cpp
-### while() used for continous loop
-### switch case used for choice of all the shapes 
-### Object is created in individual shape case 
-### Values for x & y are taken
-### Call to Plot_Corrdinate() for printing coordinate
-
-#-------------------------------------------------------------------------------------------------------------------
-
-## textfile
-### Values printed in ".txt" file 
-### Location of ".txt" file for required shape given in command of gnuplot to get expected output 
-
-#-------------------------------------------------------------------------------------------------------------------
+# GNU Plot Assigment
```

Comment:
Please put details of each feature in readme.md, from the readme file, the reader should get exact idea about project.